tfe/resource_tfe_sentinel_policy.go

```diff
@@ -125,23 +133,22 @@ func resourceTFESentinelPolicyRead(d *schema.ResourceData, meta interface{}) err
 func resourceTFESentinelPolicyUpdate(d *schema.ResourceData, meta interface{}) error {
 	tfeClient := meta.(*tfe.Client)
 
-	if d.HasChange("enforce_mode") {
-		// Create a new options struct.
-		options := tfe.PolicyUpdateOptions{
-			Enforce: []*tfe.EnforcementOptions{
-				&tfe.EnforcementOptions{
-					Path: tfe.String(d.Get("name").(string) + ".sentinel"),
-					Mode: tfe.EnforcementMode(tfe.EnforcementLevel(d.Get("enforce_mode").(string))),
-				},
+	// Create a new options struct.
+	options := tfe.PolicyUpdateOptions{
+		Description: tfe.String(d.Get("description").(string)),
```

Comment:
Since `description` is an optional field we need to check if we got a value and so if we should set this. If not we would override any manual set descriptions here.

Reply:
hmm I'm not 100% sure what you mean by this… Do you mean that, if someone has set a description for a policy in the TFE UI but does not have a description set in their provider config, the provider config should leave it alone and not replace it with `nil` or an empty string?